qa/rolling-upgrade/src/test/resources/rest-api-spec/test/mixed_cluster/20_camel_case_on_format.yml

```diff
@@ -0,0 +1,19 @@
+---
+"Verify that we can still use index with camel case date field":
+  - do:
+      bulk:
+        refresh: true
+        body:
+          - '{"index": {"_index": "camel_case_on_format"}}'
+          - '{"date_field": "2019-02-01T00:00+01:00"}'
+
+  - do:
```

Comment:
Sounds good!

Reply:
Actually I realized this is already verified by checking for 4 hits in upgraded test and since it's for the mixed cluster here, it would be random 2 or 3.

Reply:
Right, I was just thinking we'd check that the hit count is greater than 0. It's a super small comment though.